packages/components/src/table/table.js

```diff
@@ -100,11 +105,21 @@ class Table extends Component {
 
 	updateTableShadow() {
 		const table = this.container.current;
+
 		const scrolledToEnd =
 			table.scrollWidth - table.scrollLeft <= table.offsetWidth;
-		this.setState( {
-			isScrollable: ! scrolledToEnd,
-		} );
+		if ( scrolledToEnd && this.state.isScrollable ) {
```

Comment:
I think these checks are a fine way to handle things.  👍 Can we also destructure these state properties?

Reply:
👍 Done in https://github.com/woocommerce/woocommerce-admin/pull/4179/commits/6b8340e370668309dcbfccf393f28b2096730f37